static/js/hotkey.js

```diff
@@ -976,6 +977,20 @@ export function process_hotkey(e, hotkey) {
             reactions.toggle_emoji_reaction(msg.id, canonical_name);
             return true;
         }
+        case "add_first_emoji": {
+            // '=': adds to first emoji on selected message
+            // Use canonical name.
+            let add_first_emoji_code;
+            const first_reaction = reactions.get_message_reactions(msg)[0];
+            if (!first_reaction) {
+                add_first_emoji_code = "1f446";
```

Comment:
We should mention here in a comment that if there is no emoji reaction yet on the message, we add the `:this:` emoji (just like you've mentioned in the commit message)

Reply:
Let's use `:100:` instead; it seems to be more popular and a brief investigation of emojipedia suggests a meaning that's further from agreement: https://unicode-table.com/en/1F446/

Reply:
> Let's use `:100:` instead; it seems to be more popular and a brief investigation of emojipedia suggests a meaning that's further from agreement: https://unicode-table.com/en/1F446/

I've changed the emoji code (for a message with no emoji reaction yet)  to `:100:`s emoji code.